pyanaconda/install.py

```diff
@@ -60,6 +60,7 @@ def _writeKS(ksdata):
 def doConfiguration(storage, payload, ksdata, instClass):
     willWriteNetwork = not flags.flags.imageInstall and not flags.flags.dirInstall
     willRunRealmd = ksdata.realm.discovered
+    willSnapShotRoot = ksdata.autopart.autopart and ksdata.autopart.snapshot
```

Comment:
This needs to increment the number of progress steps just like the other two checks do below.